src/modules/75-cf/components/NoData/SectionNoData/SectionNoData.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { Layout, Text, FontVariation, Color } from '@harness/uicore'
+import React, { FC, PropsWithChildren } from 'react'
+import GetStartedWithFF from '@cf/components/GetStartedWithFF/GetStartedWithFF'
+import { NoData, NoDataProps } from '../NoData'
+
+export interface SectionNoDataProps extends Pick<NoDataProps, 'message' | 'description'> {
+  panels: { imageURL: string; description: string }[]
+}
+
+const SectionNoData: FC<PropsWithChildren<SectionNoDataProps>> = ({ message, description, panels, children }) => (
+  <NoData message={message} description={description}>
+    <Layout.Horizontal spacing="xxxlarge" flex={{ alignItems: 'center', justifyContent: 'center' }}>
+      {panels.map(panel => (
+        <Layout.Horizontal
+          flex={{ justifyContent: 'center', alignItems: 'center' }}
+          style={{ textAlign: 'center' }}
+          spacing="huge"
+          key={panel.description}
+        >
+          <Layout.Vertical
+            flex={{ alignItems: 'center' }}
+            key={panel.description}
+            spacing="huge"
+            padding={{ bottom: 'huge', top: 'small' }}
+          >
+            <img src={panel.imageURL} alt={panel.imageURL} width={110} height={110} />
```

Comment:
The `alt` property is for a description of the image, if applicable (for screen readers, or if it fails to load). We probably don't want the contents of the image being read out by a screen reader. Typically try to describe what's in the image. In cases like this where the image is purely decorative, a blank (but still present) `alt` is ideal. 
```suggestion
            <img src={panel.imageURL} alt="" width={110} height={110} />
```